config/initializers/assets.rb

```diff
@@ -3,9 +3,10 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 
-Rails.application.config.assets.precompile += %w( home.jpg )
-Rails.application.config.assets.precompile += %w( musikore.svg )
-Rails.application.config.assets.precompile += %w( musikore-com.svg )
+Rails.application.config.assets.precompile += %w(home.jpg)
```

Comment:
The files are in `app/assets/images` once it gave me an error asked me to add precompile. So I don't know if I really should change those lines.